ote/src/cljs/ote/app/controller/flags.cljs

```diff
@@ -0,0 +1,31 @@
+(ns ote.app.controller.flags
+  (:require [tuck.core :as tuck]
+            [ote.db.transport-service :as t-service]
+            [ote.localization :refer [tr tr-key]]
+            [clojure.string :as str]
+            [ote.communication :as comm]
+            [clojure.string :as s]))
+
+(defn true-string? [s] (= s "true"))
+
+(defn in?
+  "Return true if coll contains element"
+  [coll elm]
+  (some #(= elm %) coll))
+
+(defn is-in-production []
+  (let [page-url (-> (.-location js/window))]
+    (if (or (s/includes? (str page-url) "testi") (s/includes? (str page-url) "localhost"))
+      false
+      true)))
+
+(defn use-in-production?
+  "Check if given feature (f) is enabled in production"
+  [f]
+  (let [features  (str/split (-> js/document
```

Comment:
`(into #{} (map keyword (str/split (.getAttribute js/document.body "features") #",")))`

Reply:
If you parse it into a set, the `in?` function becomes unnecessary, just use the set as a function